docs/api/helpers.rst

```diff
@@ -1,8 +1,12 @@
+.. meta::
+  :description: hipSOLVER documentation and API reference library
+  :keywords: hipSOLVER, rocSOLVER, ROCm, API, documentation
+
 .. _api_helpers:
 
-****************
-Helper Functions
-****************
+******************
+Helper functions
```

Comment:
Shouldn't this be "hipsolver helper functions"?

Reply:
Updated